composer/algorithms/augmix/augmix.py

```diff
@@ -262,7 +265,7 @@ def apply(self, event: Event, state: State, logger: Logger) -> None:
                                     width=self.width,
                                     alpha=self.alpha,
                                     augmentation_set=self.augmentation_set)
-        assert state.dataloader is not None, "dataloader should be defined on fit start"
+        assert isinstance(state.dataloader, torch.utils.data.DataLoader), "dataloader type checked on match()"
```

Comment:
These (augmix/colout/randaugment) algorithms as written won't work with ffcv dataloader as it has no dataset. Checking with isinstance is fine here.